Assets/Xbox Live/Scripts/XboxLiveUserHelper.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Collections.Generic;
+
+public class XboxLiveUserHelper : Singleton<XboxLiveUserHelper>
+{
+
+    public List<XboxLiveUserInfo> XboxLiveUsersInTheScene { get; private set; }
```

Comment:
Yeah good point. I also will  a function for Reset that the developers can call to clear and re-initialize the XboxLiveUserManager content (to switch say between single user and multi user scenes)

---
In reply to: [119976395](https://github.com/Microsoft/xbox-live-unity-plugin/pull/66#discussion_r119976395) [](ancestors = 119976395)